source/boards/boards-settings.rst

```diff
@@ -0,0 +1,17 @@
+Settings
+========
+
+Set language
+------------
+
+To set your language on boards, select the gear icon next to your profile avatar, then go to **Set language** to apply your language settings. Please note: language settings in Boards are independent from language settings in Channels.
+
+Emoji icons
+-----------
+
+To enable or disable random emoji icons for your board and cards, select the gear icon next to your profile avatar, then toggle **Random icons on or off**.
+
+Product tour
+------------
+
+If you skipped the product tour or want a refresher on Boards, you can restart the product tour by going to the gear icon next to your profile avatar and selecting **Product tour**. This will add a new **Welcome to Boards** template to your sidebar and kick-off the guided onboarding tour.
```

Comment:
another good place to incorporate the product icon via an inline image reference